src/frontend/staff-portal/src/app/services/lookups.service.ts

```diff
@@ -214,646 +214,113 @@ export class LookupsService implements ILookupsService {
-      {
-        id: "10256.0001",
-        name: "Sparwood Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10249.0001",
-        name: "Squamish Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10267.0001",
-        name: "Stewart Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "8842.0001",
-        name: "Surrey Family Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "8841.0001",
-        name: "Surrey Provincial Court",
-        typeCode: "CTH", __team: "B"
-      },
-      {
-        id: "10239.0001",
-        name: "Tahsis Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "9072.0001",
-        name: "Terrace Law Courts",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10240.0001",
-        name: "Tofino Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10258.0001",
-        name: "Tsay Keh Dene Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10268.0001",
-        name: "Tumbler Ridge Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10241.0001",
-        name: "Ucluelet Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "9144.0001",
-        name: "Valemount Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "19228.0734",
-        name: "Vancouver Law Courts",
-        typeCode: "CTH", __team: "A"
-      },
-      {
-        id: "8813.0001",
-        name: "Vancouver Provincial Court",
-        typeCode: "CTH", __team: "A"
-      },
-      {
-        id: "10251.0001",
-        name: "Vancouver Traffic Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "9071.0001",
-        name: "Vanderhoof Law Courts",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "96.0001",
-        name: "Vernon Law Courts",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10242.0001",
-        name: "Victoria Family and Youth Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "8807.0001",
-        name: "Victoria Law Courts",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "19635.0734",
-        name: "Violation Ticket Centre",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10252.0001",
-        name: "West Vancouver Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "10243.0001",
-        name: "Western Communities Provincial Court",
-        typeCode: "CTH", __team: "D"
-      },
-      {
-        id: "9070.0001",
-        name: "Williams Lake Law Courts",
-        typeCode: "CTH", __team: "D"
-      }
+      { id: '10264.0007', name: '100 Mile House Law Courts', typeCode: "CTH", __team: 'D'},
```

Comment:
Can it store in another file ? Even required to be built to refresh it. Just want to be more clean

Reply:
Done.